components/avatar/avatar.vue

```diff
@@ -150,6 +177,30 @@ export default {
       return element?.tagName?.toUpperCase() === 'IMG';
     },
 
+    randomizeGradientAngle () {
+      const angles = [45, -45, 90, -90, 135, -135, 180, -180];
+      return getRandomElement(angles);
+    },
+
+    randomizeGradientColorStops () {
+      const colors = new Set();
+      const colorsWith100 = ['purple-100', 'magenta-100', 'gold-100', 'blue-100'];
+      const colorsWith200 = ['purple-200', 'magenta-200', 'gold-200', 'blue-200'];
+
+      // get 3 unique colors, 2 from colorsWith100 and one from colorsWith200
+      while (colors.size < 3) {
+        if (colors.size === 2) {
```

Comment:
Maybe get rid of these magic numbers, store them in descriptive constants, somethin like MAX_GRADIENT_COLORS, MAX_GRADIENT_COLORS_100